### ISSUE TITLE
Space removed after anonymous class keyword — PSR12 violation?

### ISSUE THREAD

**Issue by hackel** [prettier/plugin-php#1544]
**@prettier/plugin-php v0.14.3**
[Playground link](https://loilo.github.io/prettier-php-playground/#N4IgDgTgpgLjCWUIHkwIPYDsDOIBcok8mMA6vACYwAW+AHAAwA0IMAhgEblW14AsLAK7YoAFU648AMzYAbES2zEA5rKgBFQehhR8M+VBZhqYAGpIlWfCACsAOj4gWMCG3iyVAYXQBbH2wAFAAkA-BdBQxAOVwBjKABlGABPNWswbAgAWgAmJxBoAEdBeGgA12V-PTkFEGIRCBgytgq2KoMAXxYoCngYdBQ0eCxJUDZsGDaRTtrMMEEJvBAAHgB+YzAAHUwtgBIADwACAF4DzCgAdwOY2THsA4AKHYA3NggASgPgdoBuXaTjg5SQSYGIYTAPZ6vD5fX6YEDtIA)

**Input:**
```php
<?php

$x = new class ($var) {};
$y = function ($var) {};
```

**Output:**
```php
<?php

$x = new class($var) {};
$y = function ($var) {};
```

This issue came up due to the `PSR12.Classes.AnonClassDeclaration.SpaceAfterKeyword` rule from PHP_CodeSniffer, which marks Prettier's output as an error.  PSR-12 isn't explicit on this point, so I'm not 100% sure which is correct.  I'm assuming they based their rule off of this:

> Anonymous Classes MUST follow the same guidelines and principles as closures in the above section.

> Closures MUST be declared with a space after the function keyword, and a space before and after the use keyword.

From my perspective, it seems like PHP_CodeSniffer is correct.  If we want to be consistent, then the space should be required.

**Comment by czosel** [comment]
Thanks for reporting this @hackel! I agree, the missing space looks off. 
cc @evilebottnawi

**Comment by evilebottnawi** [comment]
If it is in PSR-12, let's fix it

**Comment by czosel** [comment]
Released as `v0.16.0` :tada:
